string[] target_label
float32[] target_x
float32[] target_y
float32[] target_w
float32[] target_h
float32 state
string[] label_point
float32[] point_x
float32[] point_y


